README.md

```diff
@@ -56,14 +56,26 @@ GUIDs provide a domain-neutral, persistent and scalable way to track data across
 
 Indexd is a two-layer system. On the bottom layer, each data object has a GUID and hashes that map to known physical locations of the data.
 
-The second layer is aliases. **Aliases** are user-defined, human-readable identifiers that map to GUIDs. This adds the flexibility of supporting human-readable identifiers and allow referencing existing identifiers that are created in other systems.
+The second layer is aliases. **Aliases** are user-defined, human-readable identifiers that map to GUIDs. This adds the flexibility of supporting human-readable identifiers and allow referencing existing identifiers (such as DOIs and ARKs) that are created in other systems.
 
 <div align="center">
 <img src="./docs/aliases.png" alt="indexd GUIDs solution" height="400" hspace="10"/>
 </div>
 
 GUIDs are primarily used to track the current location of data as it is moved or copied from one location to another. The GUID itself is at minimum 128-bits, as a [UUID](https://tools.ietf.org/html/rfc4122.html) is used as the base. Additionally, a prefix can be prepended to this UUID, which lengthens the identifier even further (this is used primarily to assist in [distributed resolution](#distributed-resolution-utilizing-prefixes-in-guids)). If you want a shorter identifier, you can use the aliases defined above to create a different, unique mapping to GUIDs.
 
+Data GUIDs are structured as follows:
+
+`dg.[prefix]/[128 UUID]`
+
+* All data guids begin with the prefix: dg
```

Comment:
that's not technically true since not all GUIDs have a prefix
maybe `All prefixes begin with dg`?